### PR TITLE
Typo fix on Maicol's testimonial

### PR DESCRIPTION
**Pull request by TurboProgramming**
This PR fixes a small typo on the testimonial of Maicol.

### CODE REVIEW

File: _data/testimonials.yml [hunk 1]
```diff
@@ -28,4 +28,4 @@
 
 - name: Maicol Battistini
   username: maicol07
-  text: Makeroid, a free platform to let your ideas become android apps. Simple, beatiful and powerful.
+  text: Makeroid, a free platform to let your ideas become android apps. Simple, beautiful and powerful.
```
Comment (conorshipp) [inline _data/testimonials.yml]: Should `android` have an uppercase A?

Comment (conorshipp) [inline _data/testimonials.yml]: Should the `a` in `android` be capitalised?

Comment (TurboProgramming) [reply _data/testimonials.yml]: @conorshipp @barreeeiroo Fixed.

File: _data/testimonials.yml [hunk 2]
```diff
@@ -28,8 +28,8 @@
 
 - name: Maicol Battistini
   username: maicol07
-  text: Makeroid, a free platform to let your ideas become android apps. Simple, beatiful and powerful.
+  text: Makeroid, a free platform to let your ideas become Android apps. Simple, beautiful and powerful.
 
 - name: Rémi Chartier
   username: R3M
-  text: Makeroid is the best way to make apps with a simple interface and simple code! There are so much components and options, and it has a beautiful Material Design
+  text: Makeroid is the best way to make apps with a simple interface and simple code! There are so much components and options, and it has a beautiful Material Design.
```
Comment (conorshipp) [inline _data/testimonials.yml]: `There are so much components` should be `There are so many components`.

**Pull request merged**